modules/odc-adding-components-to-an-existing-project.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
+
+[id="odc-adding-components-to-an-existing-project_{context}"]
+= Adding components to an existing project
+
+.Procedure
+
+. Click on the *Add to Project* (image:odc_add_to_project.png[title="Add to Project"]) or press kbd:[Ctrl+Space]
+. Search the component and select *Create* or press kbd:[Enter] to add the component in the topology.
+
+.Adding component via quick search
+image::odc_quick_search.png[]
+
+You can also use the context menu by right-clicking in the topology *Graph view* to add a service such as *Import from Git*, *Container Image*, *Database*, *From Catalog*, *Operator Backed*, *Helm Charts*, *Samples*, or *Upload JAR file*  to your project.
```

Comment:
Is this for default state with no operator installed because there are Event Source, Channel and Broker options as well if eventing CR is present

Reply:
@sanketpathak Yes, this is for the default state where there is no operator or pod installed in the topology.

Reply:
IBM SG: Use "right-click" when writing for experienced users. Use "Using the right mouse button, click..." when writing for novice users.
```suggestion
You can also use the context menu by right-clicking the topology *Graph view* to add a service such as *Import from Git*, *Container Image*, *Database*, *From Catalog*, *Operator Backed*, *Helm Charts*, *Samples*, or *Upload JAR file*  to your project.
```
Also, use "right-click", not "right-click on", just as you use "click", not "click on".

Reply:
```suggestion
You can also use the context menu by right-clicking the topology *Graph view* to add a service such as *Import from Git*, *Container Image*, *Database*, *From Catalog*, *Operator Backed*, *Helm Charts*, *Samples*, or *Upload JAR file*  to your project.
```

Reply:
to add a service such as *Import from Git*, *Container Image*, *Database*, *From Catalog*, *Operator Backed*, *Helm Charts*, *Samples*, or *Upload JAR file* --- These are not services/components. These are methods that you use to add components/services.
Also it is not clear that this is another way to do the same procedure.

Reply:
```suggestion
Alternatively, you can also use the *Import from Git*, *Container Image*, *Database*, *From Catalog*, *Operator Backed*, *Helm Charts*, *Samples*, or *Upload JAR file*  options in the context menu by right-clicking in the topology *Graph view* to add a component to your project.
```